src/frontend/components/DashboardVideoLive/index.spec.tsx

```diff
@@ -477,4 +483,27 @@ describe('components/DashboardVideoLive', () => {
     screen.getByText(/wonderful class!/i);
     screen.getByRole('textbox', { name: /description/i });
   });
+
+  it('shows the pairing button when the status is not STOPPED', () => {
+    for (const state of Object.values(liveState)) {
+      render(
+        wrapInIntlProvider(
+          wrapInRouter(
+            <Suspense fallback="loading...">
+              <DashboardVideoLive video={{ ...video, live_state: state }} />
+            </Suspense>,
+          ),
+        ),
+      );
+      if (state !== liveState.STOPPED) {
+        screen.getByTitle(`Pairing button for ${video.id}`);
+      } else {
+        const pairingSecretLabel = screen.queryByTitle(
+          `Pairing button for ${video.id}`,
+        );
+        expect(pairingSecretLabel).not.toBeInTheDocument();
+      }
+      cleanup();
+    }
+  });
```

Comment:
minor just not to have a const for the title, to ease the reading
```suggestion
  it('shows the pairing button when the status is not STOPPED', () => {
    const { getByTitle, queryByTitle } = render(
        wrapInIntlProvider(
          wrapInRouter(
            <Suspense fallback="loading...">
              <DashboardVideoLive video={{ ...video, live_state: state }} />
            </Suspense>,
          ),
        ),
      );
      if (state !== liveState.STOPPED) {
        getByTitle(`Pairing button for ${video.id}`);
      } else {
        expect(
          queryByTitle(`Pairing button for ${video.id}`),
        ).not.toBeInTheDocument();
      }
```

Reply:
Why don't using `screen` in your suggestion ? I don't see the interest to not do it.

Reply:
IMHO, @carofun suggestion is more elegant than using screen.

Reply:
just because I think nico created a const because he thought text was too long and to avoid the const

Reply:
In fact, I wrote this accordingly to the documentation : https://testing-library.com/docs/guide-disappearance/#nottobeinthedocument